README.md

```diff
@@ -32,13 +32,16 @@ the user wishes a pre-decompressed ROM may be supplied as input. Please be sure
 playing via any means other than on real N64 hardware, the use of the "Compress patched ROM" flag is strongly encouraged as uncompressed ROMs are
 impossible to inject for the Virtual Console and have random crashing problems on all emulators.
 
-For general use, there are three recommended emulators: [RetroArch](https://wiki.ootrandomizer.com/index.php?title=Retroarch), 
-[Bizhawk](https://wiki.ootrandomizer.com/index.php?title=Bizhawk) and [Project 64 (v2.4+)](https://wiki.ootrandomizer.com/index.php?title=Project64). 
-In a nutshell the differences are: Bizhawk is resource-intensive, but easy to set up and the only emulator to support 
-[Multiworld](https://wiki.ootrandomizer.com/index.php?title=Multiworld); RetroArch is less resource-intensive but can be frustrating to set up; 
-Project64 is easy to set up and the least resource-intensive, but versions compatible with OoTR are banned for races organized by our community. 
-Please follow the guides on our wiki carefully to ensure a stable game experience. For questions and tech support we kindly refer you to our 
-[Discord](https://discord.gg/q6m6kzK).
+For general use, there are three recommended emulators: [Project 64 (v2.4+)](https://wiki.ootrandomizer.com/index.php?title=Project64), 
+[Bizhawk](https://wiki.ootrandomizer.com/index.php?title=Bizhawk) and [RetroArch](https://wiki.ootrandomizer.com/index.php?title=Retroarch).
+In a nutshell the differences are: Project64 is the lightest emulator and the easiest to setup, however only v2.4+ runs OoTR well and it is also the 
+only stable version that is allowed for races; Bizhawk is the most resource-intensive, but easier to set up than RetroArch and the only race-legal emulator 
```

Comment:
"however only v2.4+ runs OoTR well and it is also the only stable version that is allowed for races" could use a bit of an improvement, I think. Maybe "however, you will need a stable version from 2.4 or later to run OoTR well (and earlier versions are not permitted for use in OoTR races)"?